.changelog/12311.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
+Forked net/rpc to add middleware support: https://github.com/hashicorp/consul-net-rpc/ .
+```
```

Comment:
I think this is an internal detail that doesn't need a changelog. Thoughts?

Reply:
@dhiaayachi wanted us to call out. At first that was my position too, that maybe it doesn't need to be in the changelog.

But I think it makes to highlight it since it's a pretty unusual dependecy update 🤷🏼